### DISCUSSION TITLE
Swiss wont use SD2SP2 as configuration device

### DISCUSSION THREAD

**Discussion by Scrimpy1999** [emukidid/swiss-gc#911] [Q&A]
I have pico boot on my GameCube and I use a 32gb micro SD with Swiss installed and my game files. It works fine but when I decide to upgrade to a 64g micro SD Swiss now won’t use the SD2SP2 to load games. It only opens my GameCube memory card. When I looked in the settings the configuration device is set to none but when I try changing it to SD2SP2 it just fails. What would be causing it to not use SD2SP2 to bring up my games?

**Comment by Extrems** [comment]
The SD card is failing to initialize.

**Reply by Scrimpy1999** [reply]
So, how would I fix the problem?

**Reply by Extrems** [reply]
There's not much you can do beside using a different SD card. The driver would need to be adapted for your SD card, if even possible.

**Reply by Scrimpy1999** [reply]
Oh, okay thanks for the info :)
